lib/compute/region.js

```diff
@@ -215,6 +221,198 @@ Region.prototype.createAddress = function(name, options, callback) {
+ *   });
+ *
+ * //-
+ * // If you anticipate many results, you can end a stream early to prevent
+ * // unnecessary processing and API requests.
+ * //-
+ * gce.getSubnetworks()
+ *   .on('data', function(subnetwork) {
+ *     this.end();
+ *   });
+ */
+Region.prototype.getSubnetworks = function(options, callback) {
+  var self = this;
+
+  if (is.fn(options)) {
+    callback = options;
+    options = {};
+  }
+
+  options = options || {};
+
+  this.request({
+    uri: '/subnetworks',
+    qs: options
+  }, function(err, resp) {
+    if (err) {
+      callback(err, null, null, resp);
+      return;
+    }
+
+    var nextQuery = null;
+
+    if (resp.nextPageToken) {
+      nextQuery = extend({}, options, {
+        pageToken: resp.nextPageToken
+      });
+    }
+
+    var subnetworks = (resp.items || []).map(function(subnetwork) {
+      var subnetworkInstance = self.subnetwork(subnetwork.name);
+      subnetworkInstance.metadata = subnetwork;
+      return subnetworkInstance;
+    });
+
+    callback(null, subnetworks, nextQuery, resp);
+  });
+};
+
+/**
+ * Get a reference to a Google Compute Engine address in this region.
+ *
+ * @resource [Instances and Networks]{@link https://cloud.google.com/compute/docs/instances-and-network}
+ *
+ * @param {string} name - Name of the subnetwork.
+ * @return {module:compute/subnetwork}
+ *
+ * @example
+ * var subnetwork = region.subnetwork('subnetwork-name');
+ */
+Region.prototype.subnetwork = function(name) {
+  return new Subnetwork(this, name);
+};
+
+/**
+ * Create a subnetwork in this region.
+ *
+ * @resource [Subnetwork Resource]{@link https://cloud.google.com/compute/docs/reference/v1/subnetworks#resource}
+ * @resource [Subnetwork: insert API Documentation]{@link https://cloud.google.com/compute/docs/reference/v1/subnetworks/insert}
+ *
+ * @param {string} name - Name of the subnetwork.
+ * @param {object} config - See a
+ *     [Subnetwork resource](https://cloud.google.com/compute/docs/reference/v1/subnetworks#resource).
+ * @param {string=} config.description - An optional description of this
+ *     resource. Provide this property when you create the resource.
+ * @param {string} config.ipCidrRange - The range of internal addresses that
+ *    are owned by this subnetwork. Provide this property when you create the
+ *    subnetwork. For example, 10.0.0.0/8 or 192.168.0.0/16. Ranges must be
+ *    unique and non-overlapping within a network.
+ * @param {string} config.network - The URL of the network to which this
+ *    subnetwork belongs, provided by the client when initially creating
+ *    the subnetwork. Only networks that are in the distributed mode can have
+ *    subnetworks.
+ * @param {string=} config.region - URL of the region where the Subnetwork
+ *    resides.
+ * @param {function} callback - The callback function.
+ * @param {?error} callback.err - An error returned while making this request.
+ * @param {module:compute/rule} callback.rule - The created Rule object.
```

Comment:
{module:compute/subnetwork} callback.subnetwork - The created Subnetwork object.